docs/Reference/API-Objects.md

```diff
@@ -219,8 +219,7 @@ Parameter for [`eth_call`](API-Methods.md#eth_call) and
 
 !!! note
 
-    All transaction call object parameters are optional for [`eth_estimateGas`](API-Methods.md#eth_estimategas).
-    Only the `to` parameter is required for [`eth_call`](API-Methods.md#eth_call).
+    All transaction call object parameters are optional for [`eth_estimateGas`](API-Methods.md#eth_estimategas) and [`eth_call`](API-Methods.md#eth_call).
```

Comment:
For simplicity, maybe delete this note and add a sentence "All fields are optional."